hooks/useTokenList.tsx

```diff
@@ -34,18 +33,18 @@ export const useTokenList = () => {
         symbol: "WETH",
         address: network.wethAddress,
         decimals: 18,
-        name: "wETH",
+        name: "Wrapped ETH",
         logoURI: "",
       };
-      const eth: TokenListItem = {
+      const baseToken: TokenListItem = {
         chainId,
-        symbol: "ETH",
+        symbol: network.baseToken.symbol,
         address: ethAddress,
         decimals: 18,
-        name: "ETH",
+        name: network.baseToken.name,
         logoURI: "",
```

Comment:
Change empty string to `network.baseToken.logoURI`?

Reply:
561c2cd